### PR TITLE
DRAFT Improve UG

### PR DESCRIPTION
**Pull request by tjingsheng**
Experimental Branch that does not work still DRAFT, do not review until completed.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -1,172 +1,191 @@
...
+    - 5.5 [Commands for notes](#55-commands-for-notes)
+        - 5.5.1 [List all notes: `note list`](#551-list-all-notes-note-list)
+        - 5.5.2 [Add a note: `note add`](#552-add-a-note-note-add)
+        - 5.5.3 [Delete a note: `note delete`](#553-delete-a-note-note-delete)
+        - 5.5.4 [Update a note: `note update`](#554-update-a-note-note-update)
+        - 5.5.5 [Tag a note: `note tag`](#555-tag-a-note-note-tag)
+        - 5.5.6 [Clear all tags from a note: `note cleartag`](#556-clear-all-tags-in-a-note-note-cleartag)
+    - 5.6 [View help : `help`](#56-view-help--help)
+    - 5.7 [Exit SEPlendid: `exit`](#57-exit-seplendid--exit)
+    - 5.8 [Save the data](#58-save-the-data)
```
Comment (lamchenghou) [inline docs/UserGuide.md:88]: Links will not work if you do this. 

As discussed, this part of the UG is assigned to me, and I have spent a long time fixing it. Please do not replace the table of contents with your own, as we are tasked to reflect contribution appropriately.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -1,172 +1,191 @@
...
 --------------------------------------------------------------------------------------------------------------------
 
 ## 1. How to navigate this User Guide?
 
-As a new user in SEPlendid, this user guide serves as an easy-to-follow guide in aiding you in executing your first 
-commands in SEPlendid, utilising SEPlendid to it's fullest potential!
+As a new user in SEPlendid, this user guide serves as an easy-to-follow guide, guiding you through the execution of your
+first commands in SEPlendid. You can unlock SEPlendid full potential in this guide!
+
+Here is a step-by-step instruction for you to navigate the user guide:
```
Comment (lamchenghou) [inline docs/UserGuide.md:102]: Are these changes really necessary? Please credit the initial writer, you only changed a few words.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -1,172 +1,191 @@
...
 
-If you are an **experienced** user, you can head over to the [Command Summary](#) section for a well-curated overview
-of the commands available in SEPlendid.
+If you are an **experienced** user, you can head over to the [Command summary](#7-command-summary) section for a
+well-curated overview of the commands available in SEPlendid.
 
-<br />
-<br />
+<br>
+<br>
```
Comment (lamchenghou) [inline docs/UserGuide.md:112]: Same here.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -637,15 +730,18 @@ Sorts universities by the university name, alphabetically. This is useful when:
...
-<br />
+
+<br>
 
 #### 5.4.1 List all mappings: `mapping list`
-Lists all available mappings from a local course offered by NUS Computing, to a partner course offered by a partner 
+
+Lists all available mappings from a local course offered by NUS Computing, to a partner course offered by a partner
 university. This is useful when:
+
```
Comment (lamchenghou) [inline docs/UserGuide.md:744]: Could you help revert on my parts of the UG? Thanks.

File: docs/UserGuide.md [hunk 5]
```diff
@@ -936,145 +1052,58 @@ welcome to update data directly by editing that data file.
...
+|                 | `sort`      | `[attribute]`                                                    |
+| `university`    | `list`      | None.                                                            |
+|                 | `search`    | `[query]`                                                        |
+|                 | `sort`      | None.                                                            |
+| `note`          | `list`      | None.                                                            |
+|                 | `add`       | `[content] [tag]`                                                |
+|                 | `delete`    | `[index]`                                                        |
+|                 | `update`    | `[index] [content]`                                              |
+|                 | `tag`       | `[index] [tag]`                                                  |
+|                 | `cleartag`  | `[index]`                                                        |
```
Comment (lamchenghou) [inline docs/UserGuide.md:1108]: Is this necessary? Personally I implemented the rowspan to allow multi-row. Here is a comparison:

image

I would suggest that we leave the current implementation be, as it accurately reflects contribution:
image

**Review by lamchenghou — CHANGES_REQUESTED** [review]
Hey Jing Sheng,

I hope you can revert on my parts of the UG, as I've noticed breaking changes, and frankly changes that do not add value.

I hope you can also be more aware in overwriting our work with such minor changes, as it will reflect in accurately on the dashboard. We're responsible for ensuring it is accurate, and might not have the time to consistently check...

File: docs/UserGuide.md (hunk 1 above)
Comment (tjingsheng) [reply docs/UserGuide.md:88]: I understand that this part is your part. This pull request was still a draft at the point of your review. The non-functioning parts of the UG is not meant to be merged. Will not be making changes to your assigned part unless it does not work. Thank you

File: docs/UserGuide.md (hunk 5 above)
Comment (tjingsheng) [reply docs/UserGuide.md:1108]: This part of the code is not meant to be merged. This pull request was still a draft at the point of your review.

File: docs/UserGuide.md (hunk 2 above)
Comment (tjingsheng) [reply docs/UserGuide.md:102]: I was trying to improve the "you" language as feedbacked by dr wee during our cs2101 tutorial. If you deem it uncessary, I shall revert

File: docs/UserGuide.md (hunk 3 above)
Comment (tjingsheng) [reply docs/UserGuide.md:112]: I wouldn't mind reverting this changes. But the links in this changes does not work. Would you like me to revert it back to you non-functioning version for you to create a PR later?

Comment (tjingsheng) [reply docs/UserGuide.md:112]: Although I do agree some of the changes in the wording is not necessary

File: docs/UserGuide.md (hunk 1 above)
Comment (lamchenghou) [reply docs/UserGuide.md]: Thanks for the clarification, let me know if there is anything amiss for the TOC. Feel free to make changes if they are minor, but I'll be checking all headers and links prior to submission. 👍

File: docs/UserGuide.md (hunk 2 above)
Comment (lamchenghou) [reply docs/UserGuide.md:102]: I agree, and also had followed Dr. Wee's kind recommendations. No need to revert. I meant above as a suggestion that you should credit the original writer if you are using a large chunk of their work. This follows after I seen that files such as `MappingAttributeEnum` (and the local/partner course equivalents) were replaced, without proper credit. Just a small note.

File: docs/UserGuide.md (hunk 3 above)
Comment (lamchenghou) [reply docs/UserGuide.md:112]: No need to revert, my concern is on authorship. Just want to make sure everyone gets credit for what they wrote.

File: docs/UserGuide.md (hunk 5 above)
Comment (lamchenghou) [reply docs/UserGuide.md]: Thanks for your clarification on this.

**Comment by tjingsheng** [comment]
This PR has gotten too messy, refer to https://github.com/AY2324S1-CS2103T-W10-2/tp/pull/263 for a neater version
